helix-term/src/commands.rs

```diff
@@ -2616,7 +2616,7 @@ pub fn command_palette(cx: &mut Context) {
                 }
             }));
 
-            let picker = Picker::new(commands, keymap, move |cx, command, _action| {
+            let picker = Picker::without_preview(commands, keymap, move |cx, command, _action| {
```

Comment:
I'd still name this new() we could also make the with_preview() a builder function that takes `self‘ so you'd chain these: new(..).with_preview(..)

Reply:
Yes, I think beside this, everything else looks okay.